.eleventy.js

```diff
@@ -84,7 +84,7 @@ const {
   serviceWorkerPartials,
 } = require(`./${transformsDir}/service-worker-partials`);
 
-module.exports = function(config) {
+module.exports = function (config) {
```

Comment:
the space in between function and parens will kill me... but i will learn to live with it.

Reply:
Lol, I'm sorry! It seemed inconsistent across the code so I didn't think anyone had a preference.